src/asciidoc/docs/develop/testingTemplates.adoc

```diff
@@ -4,66 +4,61 @@
+The additional previews include the following.
+
+- *Admin*, *About*, and *Sticker* – Display previews of standard contexts, if you've enabled the associated entitlements.
 For more on contexts, see <<withkoji-vcc-package#_context>>.
-* *Buyer receipt* and *Seller receipt* – Displays previews of the transaction receipts for the buyer and seller.
-These contexts are available if you've implemented <<withkoji-koji-iap-package#_iap,in-app purchases>> and enabled the dynamic receipts entitlement.
 
-=== Using remote previews
+- *Thumbnail* – Displays what a template looks like when shared on social media.
+The *Thumbnail* context is always enabled.
 
-The *Remote* tab provides options for previewing a template on different devices or in different environments.
+- *Receipt (seller)* and *Receipt (buyer)* – Displays previews of the transaction receipts for the buyer and seller.
+These contexts are available if you've implemented <<withkoji-koji-iap-package#_iap,in-app purchases>> and enabled the dynamic receipts entitlement.
 
-* On a mobile device – Open a preview on any device by scanning a QR code.
-Logs from the device appear in a pane in the editor.
-* In a new tab – Open a preview on a staging server.
-Logs from the tab appear in a pane in the editor.
-* With a shared link – Copy the link for the preview.
-If you share this link with someone else, they can preview the template while it's still in development.
-+
-[NOTE]
-The remote preview link will only work for as long as your editor session is still active, so it is not suitable for long-term sharing.
+* *Remote previews* - The preview pane provides options for previewing a template on different devices or in different environments.
 
-=== Testing backend routes
+- On a mobile device – Open a preview on any device by scanning a QR code.
+Click the QR code button image:QRcode.png[QR code button] in the preview pane to display the QR code.
 
-The *Routes* tab lets you test your backend routes.
-It's similar to using a program like curl or Fiddler.
-For example, you can manually enter an HTTP request, configure a header for it, send the request, and examine the results returned by the server.
+- In a new tab – Open a preview on a staging server.
+Click the remote staging button image:remote-staging.png[Remote staging button] in the preview pane to open the preview in a new tab.
 
 === Writing to the console
 
 You can test JavaScript code by using the `console.log` method, which prints to the console.
-To see the console in the Koji editor, click the console button image:consoleButton.jpg[Console button] in the *Live preview* pane.
 
-For example, the following code snippet prints *Key pressed* to the console every time you press a key, as long as the *Live preview* pane is selected.
+Although the `console.log` method can be implemented in the Koji editor, the results must be viewed in the console, which is only available when the template is run remotely, such as in a new browser tab.
```

Comment:
```suggestion
Although the `console.log` method can be implemented in the Koji editor, the results must be viewed in the console, which is only available when the template is run remotely.
```
Because it's a bit redundant with the next sentence now.